packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -59,6 +59,7 @@ function validationIssuesToLspDiagnostics(
         };
       case "UseOfDeprecatedClass":
       case "UseOfDeprecatedAggregation":
+      case "UseOfDeprecatedAttribute":
```

Comment:
the name `UseOfDeprecatedAttribute` is less specific than other names we used here.
and also not consistent.

See other names:
```
      case "UseOfDeprecatedClass":
      case "UseOfDeprecatedAggregation":
```

We can consider more specific issueKinds: ["UseOfDeprecatedField", "UseOfDeprecatedEvent", ...]. even though the implementation may be with a **single** validation function.

This could allow the end user to configure more granular severity levels for different issues (in the future, if/when we implement such a feature).